<?xml version="1.0" encoding="UTF-8"?> 
<launch>

  <node pkg="amr_x_communication" type="uwb_server.py" name="uwb_server_node_2" output="screen">
    <param name="tx_topic" value="uwb_2_tx"/>
    <param name="rx_topic" value="uwb_2_rx"/>
    <param name="serial_port" value="/dev/ttyACM1"/>
    <param name="frequency" value="20"/>
    <param name="destination" value="26395" type="int"/>
  </node>
</launch>
